plugins/module_utils/foreman_helper.py

```diff
@@ -47,6 +48,8 @@
     parameter_type=dict(default='string', choices=['string', 'boolean', 'integer', 'real', 'array', 'hash', 'yaml', 'json']),
 )
 
+parameter_ansible_spec = {k: v for (k, v) in parameter_foreman_spec.items() if k != 'id'}
```

Comment:
Is this what we used to call `foreman_spec_helper` for?

Reply:
Kinda, but it's not filtering out the `id` (and I really don't want to have that, as the API returns it, but we never set it, so it would always trigger a change).